Docs invoice default expiry

YusukeShimizu opened the pull request:
## Change Description
https://github.com/lightningnetwork/lnd/issues/7278

## Steps to Test
N/A

## Pull Request Checklist
### Testing
~~- [ ] Your PR passes all CI checks.~~
~~- [ ] Tests covering the positive and negative (error paths) are included.~~
~~- [ ] Bug fixes contain tests triggering the bug to prevent regressions.~~

### Code Style and Documentation
- [X] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [X] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
~~- [ ] Any new logging statements use an appropriate subsystem and logging level.~~
- [X]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

ellemouton reviewed (changes requested):
Thanks for the PR @YusukeShimizu! Looking good!
Please see our [contributor guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#model-git-commit-messages) for how to title commits. 

For example, I think the first commit could be titled something like: `lncli: fix addinvoice expiry flag comment` and the second one could be `lnrpc: update invoice request expiry comment` or something along those lines. 

Since this PR touches generated GRPC code, I would say that we should not skip the CI. 

It is also missing an entry in the release notes

YusukeShimizu commented:
@ellemouton 
I fixed commit messages and added release note.  

Some CIs are failed.  
I believe retry is the solution.  
Can you please tell me how to do this?

ellemouton commented on cmd/lncli/cmd_invoice.go:
```diff
@@ -57,7 +57,8 @@ var addInvoiceCommand = cli.Command{
 			Name: "expiry",
 			Usage: "the invoice's expiry time in seconds. If not " +
-				"specified an expiry of 3600 seconds (1 hour) " +
+				"specified, an expiry of " +
```
nit: could probably fit this string on just 2 lines by not wrapping so early. same for below

YusukeShimizu replied:
I understood that your suggestion is following
```suggestion
				"specified, an expiry of 86400 seconds (24 hours) is implied."
```

By this change,`line is 95 characters (lll)` is detected by go-golangci-lint.  
Could you show me any suggestion?

ellemouton replied:
looking better now! was originally spread across more lines.

ellemouton commented on lnrpc/invoicesrpc/invoices.swagger.json:
```diff
@@ -261,7 +261,7 @@
           "type": "string",
           "format": "int64",
-          "description": "Payment request expiry time in seconds. Default is 3600 (1 hour)."
+          "description": "Payment request expiry time in seconds. Default is 86400 (24 hour)."
```
nit: s/hour/hours (same for below)

ellemouton reviewed (commented):
Looking good! Left a few nits. 
Dont need to worry about the CI fails - they are known flakes.

yyforyongyu reviewed (commented):
Thanks for the PR! This should be good to go once @ellemouton 's nits are fixed!
